arboretum/permissions/README.md

```diff
@@ -95,6 +95,78 @@ day.
   from arboretum.permissions.fetchers.github.fetch_org_collaborators import GithubOrgCollaboratorsFetcher
   ```
 
+### Organization Integrity (Repository Forks)
+
+* Class: [GithubRepoForksFetcher][gh-org-fetcher]
```

Comment:
This isn't the `gh-org-fetcher`.  You should provide a proper link and reference it here correctly.

Also, it seems like it would make sense to rename the class as GithubOrgForksFetcher?  Suggest renaming the class and changing the link ref for `gh-org-fetcher` to `gh-org-collaborators-fetcher` and adding a `gh-org-forks-fetcher` link reference to button it all up.